### DISCUSSION TITLE
useHover target is not always "currentTarget" ?

### DISCUSSION THREAD

**Discussion by cedeber** [adobe/react-spectrum#2350] [Q&A] [answered]
Hello, as you can see in the screenshot bellow, the target of useHover is not always the ref element (an anchor) defined in the hook call but is sometimes one of the children (the span). Is it a known bug or do I miss something maybe? Thanks.

<img width="942" alt="Capture d’écran 2021-09-17 à 13 08 51" src="https://user-images.githubusercontent.com/3330814/133773203-79f47656-f07d-449e-a851-a777997e98ab.png">

**Comment by snowystinger** [comment, marked as answer]
Looks like we're inconsistent about that. Should probably update it, would you mind opening an issue?
Comparing https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/src/usePress.ts#L135
to https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/src/useHover.ts#L112

**Reply by cedeber** [reply]
Here we go: https://github.com/adobe/react-spectrum/issues/2355
